src/components/Editor/Editor.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { forwardRef, lazy } from "react";
+import { Extension, ReactCodeMirrorRef } from "@uiw/react-codemirror";
+import { PlaygroundState } from "@/types";
+const CodeMirror = lazy(() => import("@uiw/react-codemirror"));
+
+interface EditorProps {
+  type: PlaygroundState["selectedTab"];
+  code: string;
+  fontSize: PlaygroundState["fontSize"];
+  extensions: Extension[];
+  onChange: (newVal: string) => void;
+}
+
+type EditorComponent = React.ForwardRefExoticComponent<
+  EditorProps & React.RefAttributes<ReactCodeMirrorRef>
+>;
+
+const Editor: EditorComponent = forwardRef(
+  ({ type, code, fontSize, extensions, onChange }, ref) => {
+    const placeholder = `Insert your ${type.toUpperCase()} code here`;
+
+    return (
+      <CodeMirror
+        style={{ fontSize }}
```

Comment:
Just curious, why just fontSize?

Reply:
you mean you want to be able to edit more styling of the code editor?
this is to control the font size through the toolbar